src/test/java/racingcar/model/CarsTest.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.model;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+public class CarsTest {
+
+    @Test
+    void 입력받은_이름으로_Car객체_리스트를_만든다() {
+        MoveStrategy moveStrategy = new RandomStrategy();
+        Cars cars = new Cars(Arrays.asList("pobi", "woni", "jun"),moveStrategy);
```

Comment:
테스트를 위해서 moveStrategy의 구체 클래스를 만드는 것보다는 람다식으로 대체해볼 수 있을 것 같아요! 🤗 
학습 차원에서 `익명 클래스`로 moveStrategy를 대체해보시고 이어서 람다식으로 변경해보세요! 😃 

```suggestion
        MoveStrategy moveStrategy = new MovStrategy {
            ....
        }
        Cars cars = new Cars(Arrays.asList("pobi", "woni", "jun"),moveStrategy);
```

```suggestion
        MoveStrategy moveStrategy = new RandomStrategy();
        Cars cars = new Cars(Arrays.asList("pobi", "woni", "jun"), () -> true);
```

[참고 자료](https://tecoble.techcourse.co.kr/post/2021-11-21-functional-interface-with-lambda/)